src/boot/store.js

```diff
@@ -226,31 +226,17 @@ const migrations: { [string]: (GlobalState) => GlobalState } = {
   // `AvatarURL`.
   '18': dropCache,
 
-  // Change format of keys representing narrows, from JSON to our format.
-  '19': state => ({
+  // Change format of keys representing narrows: from JSON to our format,
+  // then for PM narrows adding user IDs.
+  '21': state => ({
     ...dropCache(state),
-    drafts: objectFromEntries(
-      // Note this will migrate straight to our current format, even after
-      // that changes from when this migration was written!  That saves us
-      // from duplicating `keyFromNarrow` here... but calls for care in
-      // migrations for future changes to `keyFromNarrow`.
-      Object.keys(state.drafts).map(key => [keyFromNarrow(JSON.parse(key)), state.drafts[key]]),
```

Comment:
Looks like `keyFromNarrow` and `objectFromEntries` need their imports removed (but then don't forget to bring back `objectFromEntries`'s for migration 22 🙂).